src/convnets/unet.jl

```diff
@@ -0,0 +1,124 @@
+function PixelShuffleICNR(inplanes, outplanes; r = 2)
+	return Chain(basic_conv_bn((1, 1), inplanes, outplanes * (r^2)),
```

Comment:
```suggestion
    return Chain(basic_conv_bn((1, 1), inplanes, outplanes * (r^2)),
```
Metalhead uses a single tab only. You might need to configure JuliaFormatter to follow the same.